AutoCollection/Statsbeat.ts

```diff
@@ -148,15 +164,52 @@ class Statsbeat {
         this._retryCount++;
     }
 
-    public async trackStatsbeatMetrics() {
+    public async trackShortIntervalStatsbeats() {
+        this._getResourceProvider(async () => {
+            let networkProperties = {
+                "os": this._os,
+                "rp": this._resourceProvider,
+                "cikey": this._cikey,
+                "runtimeVersion": this._runtimeVersion,
+                "language": this._language,
+                "version": this._sdkVersion,
+                "attach": this._attach,
+                "instrumentation": this._instrumentations,
+            }
+            this._trackRequestDuration(networkProperties);
+            this._trackRequestsCount(networkProperties);
+            let attachProperties = {
+                "os": this._os,
+                "rp": this._resourceProvider,
+                "rpid": this._resourceIdentifier,
+                "cikey": this._cikey,
+                "runtimeVersion": this._runtimeVersion,
+                "language": this._language,
+                "version": this._sdkVersion
+            }
+            this._statbeatMetrics.push({ name: Constants.StatsbeatCounter.ATTACH, value: 1, properties: attachProperties });
+            await this._sendStatsbeats();
+        });
+    }
+
+    public async trackLongIntervalStatsbeats() {
         this._getResourceProvider(async () => {
-            this._trackRequestDuration();
-            this._trackRequestsCount();
+            let featureProperties = {
+                "os": this._os,
+                "rp": this._resourceProvider,
+                "cikey": this._cikey,
+                "runtimeVersion": this._runtimeVersion,
+                "language": this._language,
+                "version": this._sdkVersion,
+                "attach": this._attach,
+                "feature": this._features,
```

Comment:
@heyams @hectorhdzg Do we have a comprehensive list of all the `features` we will be tracking for each language? It would also be good to coordinate bit representations for common features across our sdks.

Reply:
each language will have their own specific features..  tracking AAD enablement will be a common feature.. other than that, there is nothing else i can think of.

Reply:
What about local storage? Standard metrics? Pre-aggregated metrics? Live metrics? All of these are/will be features that will be supported by all sdks so we should agree on a list.

Reply:
@lzchen having a shared list of features will make easier the data consumption here, happy to add them once we agree on them

Reply:
Statsbeat is designed only for Breeze endpoints. Live metrics is not tracked. Java hasn't implemented pre-aggregated metrics and standard metrics yet.  
I am planning to create a new set of statsbeat metrics for Local Storage.  As of now, AAD is the only common feature.  Let's get the basic Statsbeat done first and we can add the rest as Statsbeat evolves.. A list to agree upon at this moment is premature.  Please document it somewhere for future consideration.  (cc @ramthi)